src/main/java/org/jabref/gui/FXDialogService.java

```diff
@@ -51,18 +58,28 @@
  */
 public class FXDialogService implements DialogService {
 
+    private static final Duration TOAST_MESSAGE_DISPLAY_TIME = Duration.millis(3000);
+    private static final Logger LOGGER = LoggerFactory.getLogger(FXDialogService.class);
+
     private final Window mainWindow;
+    private final JFXSnackbar statusLine;
 
     /**
      * @deprecated try not to initialize a new dialog service but reuse the one constructed in {@link org.jabref.gui.JabRefFrame}.
      */
     @Deprecated
     public FXDialogService() {
-        this(null);
+        this(null, null);
```

Comment:
The `null` argument will lead to a NPE below in `notify`. Leaving it as `this(null)` should work.

Reply:
Ok, I will fix it in the next commit.